native/dll/DShowCapture-2/GraphManager.cpp

```diff
@@ -133,15 +137,26 @@ JNIEXPORT jlong JNICALL Java_sage_DShowCaptureDevice_initGraph0
 		rvs[1] = CreateDshowCaptureInfo( env, videoCaptureFilterName, capDevNum, deviceCaps, tvType, 
 			                                 TuningMode, Country );
 
+		strncpy(TuningMode, GetBDAType(deviceCaps, 2), sizeof(TuningMode));
+		slog((env, "Create first DshowCapture %s.\r\n", TuningMode));
+		rvs[0] = CreateDshowCaptureInfo(env, videoCaptureFilterName, capDevNum, deviceCaps, tvType,
+			TuningMode, Country);
+
+
+		strncpy(TuningMode, GetBDAType(deviceCaps, 3), sizeof(TuningMode));
+		slog((env, "Create second DshowCapture %s.\r\n", TuningMode));
+		rvs[1] = CreateDshowCaptureInfo(env, videoCaptureFilterName, capDevNum, deviceCaps, tvType,
+			TuningMode, Country);
+
```

Comment:
If you understand what the below block is doing can you add a comment to that effect? I'm not really following what this did before...or what the changes are modifying. This was some confusing code. :)

Reply:
Don't understand how it works but I can see there's a couple of copy/paste errors and a bit missing in the commit.  Looks like it's creating the ?table? to be used for AddBDAVideoCaptureFilters later in the code
Also a bit disconcerting all the //ZQ REMOVE ME comments through the code